### PR TITLE
Fixes scoping bug with voiceEmitter const

### PR DESCRIPTION
**Pull request by tdonia**
My apologies, @ohtangza & team -- I didn't initialize the voiceEmitter const outside the if block in #66 meaning it wasn't be set to the proper scope.  This PR fixes that bug.  My apologies for not testing more completely.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -8,8 +8,9 @@ import React, {
 const { Voice } = NativeModules;
 
 // NativeEventEmitter is only availabe on React Native platforms, so this conditional is used to avoid import conflicts in the browser/server
+const voiceEmitter = null;
 if (Platform.OS !== "web") {
-  const voiceEmitter = new NativeEventEmitter(Voice);
+  voiceEmitter = new NativeEventEmitter(Voice);
```
Comment (ohtangza) [inline index.js]: `const` variable cannot be reassigned. Isn't it causing any error on RN side?

Comment (ohtangza) [inline index.js]: `const` variable cannot be reassigned. Isn't it causing any error on RN side?

**Comment by tdonia** [comment]
@ohtangza -- updated to use `let` instead.  oddly enough it wasn't causing an error for me with const but you're absolutely right, that should not work.

**Comment by misino** [comment]
Maybe  just not important detail, but you can avoid using `let` by writing it this way:
`const voiceEmitter = Platform.OS !== "web" ? new NativeEventEmitter(Voice) : null;`

**Comment by misino** [comment]
I also spotted that `voiceEmitter` is used in method `start` without any null check.
Would you add an if condition there?
`if (voiceEmitter) voiceEmitter.addListener...`

**Comment by tdonia** [comment]
Thanks @misino -- let me know if that looks good!

**Comment by ohtangza** [comment]
@misino Could you review the last change?

**Comment by misino** [comment]
@wenkesj Would you release new bugfix version after merging this PR? I think that there were 4-5 other fixes done since last release. :)

**Comment by wenkesj** [comment]
@misino yeah sure, I’ll do some review really quick!

**Comment by wenkesj** [comment]
@misino @tdonia @ohtangza for the short term, this seems logical. 

The future possibility would be to add a dev-level warning for the web platform (or adding support for the native web platform using HTML5 Audio APIs). It may not be clear enough for users directly porting this to the web platform.

**Pull request merged**